duckduckgo/Android
# Update app dependencies

Task/Issue URL: https://app.asana.com/0/608920331025315/1169361282385305/f
Tech Design URL: 
CC: 

**Description**:
Updated most of the libraries which had newer versions available. A full list (along with some of the highlights of the new versions available in the linked Asana task)

**Steps to test this PR**:
1. This requires **thorough** manual testing of the whole app. Collectively, these libraries have a large footprint of potential change and therefore we need to ensure the whole app is behaving (and looking) as we'd expect it to.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
